src/fu-plugin-vfuncs.h

```diff
@@ -70,6 +70,9 @@ gboolean	 fu_plugin_update_prepare		(FuPlugin	*plugin,
 gboolean	 fu_plugin_update_cleanup		(FuPlugin	*plugin,
 							 FuDevice	*dev,
 							 GError		**error);
+gboolean	 fu_plugin_device_added_usb		(FuPlugin	*plugin,
```

Comment:
hmm maybe `fu_pluglin_usb_device_added` (and the corresponding other names)?

Reply:
Agree, will fix.